lib/core/util.c

```diff
@@ -828,9 +828,12 @@ void h2o_add_server_timing_header(h2o_req_t *req)
     h2o_iovec_t dst = {NULL};
 
 #define LONGEST_STR                                                                                                                \
-    ELEMENT_LONGEST_STR(connect)                                                                                                   \
-    DELIMITER ELEMENT_LONGEST_STR(header) DELIMITER ELEMENT_LONGEST_STR(body) DELIMITER ELEMENT_LONGEST_STR(request_total)         \
-        DELIMITER ELEMENT_LONGEST_STR(process) DELIMITER ELEMENT_LONGEST_STR(response)
+    ELEMENT_LONGEST_STR("connect")                                                                                                 \
+    DELIMITER ELEMENT_LONGEST_STR("header") DELIMITER ELEMENT_LONGEST_STR("body") DELIMITER ELEMENT_LONGEST_STR("request_total")   \
+        DELIMITER ELEMENT_LONGEST_STR("process") DELIMITER ELEMENT_LONGEST_STR("proxy-idle")                                       \
+            DELIMITER ELEMENT_LONGEST_STR("proxy-connect") DELIMITER ELEMENT_LONGEST_STR("proxy-request-header")                   \
+                DELIMITER ELEMENT_LONGEST_STR("proxy-request-body") DELIMITER ELEMENT_LONGEST_STR("proxy-request-total")           \
+                    DELIMITER ELEMENT_LONGEST_STR("proxy-first-byte")
```

Comment:
Do these field names match those that are actually being emitted?

Reply:
oops, thank you for finding this. Fixed at https://github.com/h2o/h2o/pull/1717/commits/06b4a9bbf24aa8310dd27db8f7b12a226952855e
And also I found similar bug exists in h2o_build_server_timing_trailer, then I fixed it at https://github.com/h2o/h2o/pull/1717/commits/ce33863ffeb361bae528af629251d829e95cec65